core/src/mapped_membank.h

```diff
@@ -0,0 +1,36 @@
+// Copyright 2018 Robin Linden <dev@robinlinden.eu>
```

Comment:
We should use `Copyright 2018 Evil Corp Ltd. contributors` or something like it.